### DISCUSSION TITLE
[polyControl] Could we provide further ability to fire an event which specific the moving point operation?

### DISCUSSION THREAD

**Discussion by ritali4912** [fabricjs/fabric.js#9503] [Q&A] [answered]
It is great that there is polyControl provides the action handler for moving a point.
Could we provide further ability to fire an event which specific the moving point operation?

I saw we have 'wrapWithFireEvent' but:

1. By default, 'createPolyActionHandler' does not use it. Of course, we can override in local. But it will be better if support it by default.
2.  It only can fire TModificationEvents (moving, scaling, rotating, skewing and resizing), that means the moving object and moving point share the same event - 'object:moving'. 
Is it possible we separate the 'moving point' from 'moving object'?

Thanks.

**Comment by ShaMan123** [comment]
Ahh it doesn't? We missed that then. It should. Let's PR

**Reply by ShaMan123** [reply, marked as answer]
#9504
